.gitignore

```diff
@@ -1,6 +1,6 @@
 .build*
-istio-*.tar.gz
```

Comment:
Likely not the PR for this, but can we change this to *.tar.gz, *.tar.xz, and *.src.rpm rather than flagging specific tar.gz files?

Reply:
Definitely not the PR for this since mergify merged it. Lol.

Reply:
sure we can.